samples/tab-user-profile/ts/src/components/home.jsx

```diff
@@ -0,0 +1,35 @@
+// <copyright file="home.jsx" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+// </copyright>
+
+import React from "react";
+import './home.css';
+
+// Mail Module in Teams Outlook default page home.tsx
```

Comment:
// User Profile Module in Teams Outlook default page home.tsx

Reply:
Fixed